opencensus/trace/ext/flask/flask_middleware.py

```diff
@@ -244,10 +250,28 @@ def get_flask_header():
     :rtype: str
     :returns: Trace context header in HTTP request headers.
     """
-    header = flask.request.headers.get(_FLASK_TRACE_HEADER)
+    return get_flask_headers([_FLASK_TRACE_HEADER]).get(_FLASK_TRACE_HEADER)
 
-    # In case the header is unicode, convert it to string.
-    if header is not None:
-        header = str(header.encode('utf-8'))
 
-    return header
+def get_flask_headers(header_names):
```

Comment:
Same as `get_django_header`, we don't need a separate method for `get_flask_header`.